src/utils.ts

```diff
@@ -184,3 +184,24 @@ export const getAppMetadata = (
             appToken,
         },
     });
+
+type ActionBehavior = {
+    behavior: 'confirm' | 'default' | 'cancel';
+    title?: string;
+    message?: string;
+    acceptText?: string;
+    cancelText?: string;
+};
```

Comment:
If `title`, `message`, `acceptText` and `cancelText` are mandatory with behavior `'confirm'`, we could make this type more accurate:
```typescript
type ActionBehavior =
    | {
          behavior: 'confirm';
          title: string;
          message: string;
          acceptText: string;
          cancelText: string;
      }
    | {
          behavior: 'default';
      }
    | {
          behavior: 'cancel';
      };
```

Reply:
nice!